pkg/installer/generateconfig.go

```diff
@@ -110,7 +110,12 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 		masterZones = []string{""}
 	}
 
-	SoftwareDefinedNetwork := string(api.SoftwareDefinedNetworkOpenShiftSDN)
+	// Set NetworkType to OVNKubernetes by default
+	SoftwareDefinedNetwork := string(api.SoftwareDefinedNetworkOVNKubernetes)
```

Comment:
This should be `softwareDefinedNetwork` , capitalized names are for exported variables/functions

Reply:
@facchettos can you put in a PR for it?  I'll cherry-pick it onto the release branches.